CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+23.09.0
+* Deprecated the init time configuration of 'setEventsBufferSize(eventsBufferSize)'. Introduced replacement 'setEventQueueSizeToSend(eventsQueueSize)
+* Deprecated the init time configuration of 'setSendUpdateEachSeconds(sendUpdateEachSeconds)'. Introduced replacement 'setUpdateSessionTimerDelay(delay)
+

```

Comment:
I think this is nice 👍  ( ' ) missing at the end for both line though. 
What do you guys think about showing the parameters ? mb unnecessary?

Reply:
yup, the ( ' ) are missing.
I feel like showing function parameters is a nice touch and probably for the best

Reply:
got it